src/items/traits.md

```diff
@@ -110,7 +110,7 @@ trait TraitMethods {
 ```
 
 ```rust,compile_fail
-// This trait is object-safe, but these methods cannot be dispatched on a trait object.
```

Comment:
You should double-check if there are any other mentions of `/(non[\- _])?obj(ect)?[\- _](un)?safe(ty)?/i`. and update them accordingly.